e2e/cypress/support/api/system.js

```diff
@@ -54,6 +54,24 @@ Cypress.Commands.add('apiUploadLicense', (filePath) => {
     cy.apiUploadFile('license', filePath, {url: '/api/v4/license', method: 'POST', successStatus: 200});
 });
 
+Cypress.Commands.add('apiInstallTrialLicense', () => {
+    const NUMBER_OF_USERS = 100;
+
+    return cy.request({
+        headers: {'X-Requested-With': 'XMLHttpRequest'},
+        url: '/api/v4/trial-license',
+        method: 'POST',
+        body: {
+            trialreceive_emails_accepted: true,
+            terms_accepted: true,
+            users: NUMBER_OF_USERS,
+        },
+    }).then((response) => {
+        expect(response.status).to.equal(200);
+        return cy.wrap(response.body);
+    });
```

Comment:
I see that the email used is the one from the currently logged in user. Is that email constant in all runs of the tests? I think that is what @Adovenmuehle needs to know.

Reply:
Thanks for checking. Yeah, it's like that but the one can successfully request will be user with manage_system permission, and we're only limiting the request to `sysadmin`. The email is `'sysadmin@sample.mattermost.com'` https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/support/api/user.js#L149.

Reply:
Make sure you let me know if that email ever changes.

Reply:
Sure Alex, will do.